sdk/nodejs/package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "@pulumi/zeet-native",
+    "name": "@zeet-dev/pulumi",
```

Comment:
actually, it should probably be `pulumi-zeet-native`

Reply:
i'll change it to pulumi-zeet. i looked like bunch of packages on pulumi registry it seems like they prefer to have the "-native" suffix be truely native which means native package gets pulumi-zeet and non-native package gets pulumi-zeet-tf or something like that

Reply:
good find